transformations/gender_randomizer/README.md

```diff
@@ -0,0 +1,45 @@
+# Gender Randomizer 🦎  + ⌨️ → 🐍
+
+Author name: Tabitha Sugumar
+Author email: t.sugumar@elsevier.com
+Author Affiliation: Elsevier
+
+## What type of a transformation is this?
+This transformation changes names in English texts, randomizing selection so there's an even chance of male and female names. It modifies pronouns to match the selected name.
+
+## What tasks does it intend to benefit?
+This is intended to avoid gender bias in natural language processing models. Run this transformation on text data prior to using it to train a model.
+
+## Previous Work
+This uses the coreferee library (https://github.com/msg-systems/coreferee). The code is downloaded and included locally, to allow for slight modifications to the setup file to allow for installation in python 3.7, as required for this transformations (the coreferee library was designed/tested in python 3.8).
+
+The names directory comes from https://www.kaggle.com/nltkdata/names. The README within the directory provides more detail.
+
+## What are the limitations of this transformation?
+This transformation does not handle gendered words such as actor/actess, waiter/waitress, etc. The handling of pronouns is limited to what the coreferee library can identify, and, in this version is limited to the segment of text fed in one iteration (ie: if a text is separated and fed in batches, the pronouns/names will not be consistent accross the text).
+
+## Examples of this transformation
+
+Because this is a randomized transformation, in both the selection of gender and selection of name, test examples are impossible -- the output for a single sentence is expected to be different in each successive run. Instead I've provided some example sentences and outputs for reference.
```

Comment:
I believe you can use a default seed in the argument in `init` of your GenderRandomizer transformation so you can generate consistent results for your test cases so you can include them in your test.json

Quite a few of the PRs use this approach for test cases.

See for example:
https://github.com/GEM-benchmark/NL-Augmenter/pull/164/files

Reply:
Thanks Timothy! When I tried this, the same name was predicted for each sentence, so for use as intended the user would have to modify the code after downloading. Should I still go ahead and do this?

Reply:
Hi Timothy, I added in the seed in the initializer, the name names does get predicted each time though, I hope it's ok! Test cases are also added in the test.json